sentry-sample/src/main/java/io/sentry/sample/NativeSample.java

```diff
@@ -6,6 +6,8 @@
   public static native void message();
 
   static {
+    System.loadLibrary("log");
+    System.loadLibrary("sentry");
```

Comment:
@Swatinem why do we need this here?
this should be handled exclusively by `NdkIntegration` and clients should not need extra code.

Reply:
it'd work without this btw, so I think its just a copypasta and can be removed, right?

Reply:
I guess it can? Is the execution order of these `static` blocks guaranteed in some way?

Reply:
@Swatinem this is the sample and not part of the SDK -> `sentry-sample` package.
`SentryNdk` is the class responsible for initing sentry-native.
not sure why you've have added these 2 lines here, mind explaining?

Reply:
well the sample only loads its own library, so I thought I load the transitive dependencies as well. I guess this is irrelevant when sentry-android runs *first* to load them.

Reply:
yep, exactly